### ISSUE TITLE
Repo at publiccodenet

### ISSUE THREAD

**Issue by jgroenen** [standard-for-public-code/standard-for-public-code#1032]
Should we archive https://github.com/publiccodenet/standard or make it clear this will be a downstream version of https://github.com/standard-for-public-code/standard-for-public-code/ from now?

**Comment by ElenaFdR** [comment]
Definitely, except I don't think any of the old staff still have rights to @publiccodenet repos.

**Comment by clausmullie** [comment]
My sense is that the Foundation will not archive the repo, as they will likely keep a Foundation branded version of the standard.

But it would be nice to update that readme, pointing to this repo as upsteam.

**Comment by ElenaFdR** [comment]
If someone wants to make a PR to update the README, then I can chat it through with @nebster and offer to merge it.

**Comment by janderssonse** [comment]
https://github.com/publiccodenet/standard/pull/3 - check if this small addition is fine @ElenaFdR
